recipes/airss/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "airss" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://www.mtg.msm.cam.ac.uk/files/airss-{{ version }}.tgz 
+    sha256: b54c61b2988b3e6ee6d01e8b73fc60d76f36ad47eeb459d1a0358e486195cc13
+    patches:
+      - patches/makefile.patch
+  - url: https://www.mtg.msm.cam.ac.uk/files/symmol.zip
+    sha256: 5c42701ff54b9c975fe2fff4e9838ef84f14b46a791ff32297558f231ae3e1c3
+    folder: external/symmol
```

Comment:
Is this vendoring symmol?

Reply:
Yes, I haven't found a feedstock on conda-forge providing it.